homeassistant/components/renson/fan.py

```diff
@@ -41,6 +44,39 @@
     Level.LEVEL4.value: 4,
 }
 
+SET_TIMER_LEVEL_SCHEMA = {
+    vol.Required("timer_level"): vol.In(
+        ["level1", "level2", "level3", "level4", "holiday", "breeze"]
+    ),
+    vol.Required("minutes"): cv.positive_int,
+}
+
+SET_DAY_NIGHT_TIME_SCHEMA = {
+    vol.Required("day"): cv.time,
+    vol.Required("night"): cv.time,
+}
+
+
+SET_BREEZE_SCHEMA = {
+    vol.Required("breeze_level"): vol.In(["level1", "level2", "level3", "level4"]),
+    vol.Required("temperature"): cv.positive_int,
+    vol.Required("activate"): bool,
+}
+
+SET_POLLUTION_SETTINGS_SCHEMA = {
+    vol.Required("day_pollution_level"): vol.In(
+        ["level1", "level2", "level3", "level4"]
+    ),
+    vol.Required("night_pollution_level"): vol.In(
+        ["level1", "level2", "level3", "level4"]
+    ),
+    vol.Optional("humidity_control"): bool,
+    vol.Optional("airquality_control"): bool,
+    vol.Optional("co2_control"): bool,
+    vol.Optional("co2_threshold"): cv.positive_int,
+    vol.Optional("co2_hysteresis"): cv.positive_int,
```

Comment:
Is there a specific reason you only set default values in the frontend(`services.yaml`) and not in the schema?

Reply:
Added default values to the schema